stablehlo/transforms/Passes.td

```diff
@@ -141,6 +141,11 @@ def StablehloAggressiveSimplificationPass
   ];
 }
 
+def StablehloConvertToSignlessPass : Pass<"stablehlo-convert-to-signless", "ModuleOp"> {
+  let summary = "Pass to transform the IR to be on signless integers.";
+  let dependentDialects = ["mlir::stablehlo::StablehloDialect"];
```

Comment:
I don't think we need this since we aren't explicitly calling any StableHLO op build methods, using upstream generic builders for everything (MHLO's pass doesn't register a dependent dialect).